src/testsuite/vec_perf_f64.c

```diff
@@ -75,8 +76,9 @@ int timed_is_f64 (void)
 
 int timed_fpclassify_f64 (void)
 {
-#ifndef __clang__
-// Clang does not define vec_test_data_class and friends.
+#ifndef PVECLIB_DISABLE_F128MATH
+/* Disable for __clang__ because of bug involving <math.h>
+   in combination with -mcpu=power9 -mfloat128 */
```

Comment:
delete this comment?
Is this comment correct? The comment (about fpclassify) does not seem to match the ifdef (about Float128).

Reply:
OK